generator/config.go

```diff
@@ -1,8 +1,48 @@
 package generator
 
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
 type PluginsConfigs map[string]interface{}
 type GenerateConfig struct {
-	GenerateTraces bool   `yaml:"generate_tracer"`
-	VendorPath     string `yaml:"vendor_path"`
-	PluginsConfigs `yaml:",inline"`
+	GenerateTraces        bool     `yaml:"generate_tracer"`
+	VendorPath            string   `yaml:"vendor_path"`
+	Imports               []string `yaml:"imports"`
+	PluginsConfigsImports []PluginsConfigs
+	PluginsConfigs        `yaml:",inline"`
+}
+
+func (gc *GenerateConfig) ParseImports() error {
+	for _, importPath := range gc.Imports {
+		normalizedPath := os.ExpandEnv(importPath)
+		normalizedPath, err := filepath.Abs(normalizedPath)
+		if err != nil {
+			return errors.Wrapf(err, "failed to make normalized path '%s' absolute", normalizedPath)
+		}
+
+		cfgFile, err := os.Open(normalizedPath)
+		if err != nil {
+			return err
+		}
+		cfg, err := ioutil.ReadAll(cfgFile)
+		if err != nil {
+			return err
```

Comment:
you can use `ioutil.ReadFile` instead of open and read it separately

Reply:
done, thanks